pkgs/top-level/python-packages.nix

```diff
@@ -21943,13 +21943,13 @@ EOF
   };
 
   libvirt = let
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
ok
I renamed the python package from ```libvirt``` to ```libvirt-python``` to avoid naming conflict with the c library it depends which resulted in infinite recursion